pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -1099,6 +1141,29 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 			}
 		}
 	}
+	// append skipped volumes
+	backup.Status.Volumes = append(backup.Status.Volumes, skipVolInfo...)
+	// add for fail and partial success
+	if len(backup.Status.Volumes) == len(skipVolInfo) {
+		// This case signifies that none of the volumes are successfully backed up
+		// hence marking it as failed
+		partialFailed = true
+	} else {
+		partialSuccess = true
+	}
+	if !partialSuccess && partialFailed {
```

Comment:
nit You just need to check !partialSuccess.  Is & PartialFailed necessary? I am assuming you are any way skipping the skipdriver volumes.. so for those volumes partialSuccess will never be marked true and for others even one volume is successful partialSuccess will be true.